src/providers/TransactionBuilderContextProvider.tsx

```diff
@@ -80,14 +80,6 @@ const TransactionBlockListItemWrapper = styled.div<{ disabled?: boolean }>`
   &:last-child {
     margin-bottom: 0;
   }
-
-  ${({ disabled }) =>
-    !disabled &&
-    `
-    &:hover {
```

Comment:
no longer any indication for hover?

Reply:
Yeah it was mentioned in the ticket, to remove underline on hover